workflow/deploy.md

```diff
@@ -0,0 +1,150 @@
+# Deploy to Kubernetes
+
+This section will present how to deploy a new service to our instance of Kubernetes.
+
+*To deploy a new website you need to be sure that you are **not** using `bower` and the the run script uses the version **1.** of yarn.*
+
+## Create a dockerfile of your service
+To deploy you service to Kubernetes you need to have an image of your project. You can have a look in the Dockerfile of the [www.ubuntu.com](https://github.com/canonical-websites/www.ubuntu.com/blob/master/Dockerfile)
```

Comment:
Could we link to https://docs.docker.com/engine/reference/builder/ somewhere in this block of text? In case people don't know what a dockerfile is.